jmix-security/security-flowui/src/main/java/io/jmix/securityflowui/view/resetpassword/UiExportHelper.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2024 Haulmont.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.jmix.securityflowui.view.resetpassword;
+
+import com.vaadin.flow.component.grid.Grid;
+import io.jmix.core.annotation.Internal;
+import io.jmix.flowui.component.grid.DataGrid;
+import io.jmix.flowui.download.Downloader;
+import io.jmix.gridexportflowui.GridExportFlowuiConfiguration;
+import io.jmix.gridexportflowui.exporter.ExportMode;
+import io.jmix.gridexportflowui.exporter.excel.ExcelExporter;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnClass;
+import org.springframework.stereotype.Component;
+
+/**
+ * Helper that provides grid exporter.
+ */
+@Internal
+@Component("sec_ExportUiHelper")
+@ConditionalOnClass(GridExportFlowuiConfiguration.class)
+public class UiExportHelper {
```

Comment:
Indeed, the exception was related to the fact that the `sandbox` project has a `reports-add-on` added to it. `reports-add-on` carried a transitive dependency on `gridexport-add-on`. Thus, the `gridexport` configuration class got into the resulting class-path. Because of this, the `UiExportHelper` bean tried to initialize, while the `gridexport` configuration was absent in the spring context.

Fixed by using `@ConditionalOnBean`